ci/requirements.txt

```diff
@@ -0,0 +1,18 @@
+dateutil
+pytz
+openpyxl
+xlsxwriter
+xlrd
+html5lib
+patsy
+beautiful-soup
+numpy
+cython
+scipy
+numexpr
+pytables
+matplotlib
+lxml
+sqlalchemy
+bottleneck
+pymysql
```

Comment:
`psycopg2` is not available in the main anaconda channels for windows. So including it will give an error anyway for the people on windows